tests/__mocks__/@myparcel/js-sdk/dist/data/delivery-options/getDeliveryOptionsEntry.js

```diff
@@ -1,3 +1,45 @@
+const morningEveningShipmentOptions = [
+  {
+    name: 'large_format',
+    schema: {
+      type: 'boolean',
+      enum: [
+        true,
```

Comment:
Waarom is er een enum hier nodig?

Reply:
Dit is een api response van /delivery-options